django/template/base.py

```diff
@@ -401,7 +419,7 @@ def create_token(self, token_string, position, lineno, in_tag):
 class DebugLexer(Lexer):
     def _tag_re_split_positions(self):
         last = 0
-        for match in tag_re.finditer(self.template_string):
+        for match in self.tag_re.finditer(self.template_string):
```

Comment:
```suggestion
        # RemovedInDjango70Warning: When the deprecation ends, replace with:
        # for match in self.tag_re.finditer(template_string):
        for match in self.tag_re.finditer(self.template_string):
```
Tried to give some examples, but essentially we need comments to explain how we should update the code when the deprecation ends. Would also need some comments within tests (essentially if we follow the instructions and make the updates, the tests should past)